app/sequential.py

```diff
@@ -2,28 +2,13 @@
 import re
 from base64 import b64decode
 from datetime import datetime
+from caldera.app.utility.planner_base import PlannerBase
 
 
-class LogicalPlanner:
+class LogicalPlanner(PlannerBase):
```

Comment:
i like where you're head is at on this-- in a normally structured application, this would be perfect. in our current design, it breaks our unspoken (or only spoken?!) 1-way contract from the core code -> plugins. this means we can't import core code inside a plugin (at least our dev supported ones). we should rely on bucketing these as services instead. i could see this becoming a planning_svc, which the planners can dynamically import functions from (or use in some capacity).